src/node/rpc/memberfrontend.h

```diff
@@ -365,6 +366,24 @@ namespace ccf
         return jsonrpc::success(true);
       };
       install(MemberProcs::UPDATE_ACK_NONCE, update_ack_nonce, Write);
+
+      // Add a new node
+      auto add_node = [this](RequestArgs& args) {
+        NodeInfo new_node = args.params;
+#ifdef GET_QUOTE
```

Comment:
I'm confused as to why we need to verify the quote here. Don't we verify it on join anyway?